### DISCUSSION TITLE
Timeline/filter for the map under Places

### DISCUSSION THREAD

**Discussion by tiefschneekamerad** [photoprism/photoprism#3319] [Ideas]
What would be a great feature to have, is a timeline, or a time filter in the map under Places. This way one could quickly filter photos that were taken at the same place but at a different time. Yearly or even weekly events could then be very quickly narrowed down.

**Comment by lastzero** [comment]
I agree with that, but I don't know when we will get to it. For now, you can use most of the search filters in the main search to limit what is shown on the map, for example `year:2023` to show only pictures from this year.

**Reply by tiefschneekamerad** [reply]
that is a good hint with the year. Thank you very much!

**Reply by lastzero** [reply]
Most of these should also work when you use the search in Places:
- https://www.photoprism.app/kb/search-filters

**Reply by tiefschneekamerad** [reply]
it works great to narrow down a single day, but is there a way to narrow it down to lets say a week: year:2022 month:6 day:13-17? The "|" that is mentioned works as logical or?

**Reply by lastzero** [reply]
See the overview. Before and after should work for this.

**Reply by tiefschneekamerad** [reply]
Fantastic! Thank you very much! Will setup photoprism tomorrow on my truenas system!
